sdcm/cluster.py

```diff
@@ -727,6 +729,32 @@ def destroy(self):
         self.stop_nemesis()
         super(ScyllaCluster, self).destroy()
 
+    def update_db_binary(self, node_list=None):
+        if node_list is None:
+            node_list = self.nodes
+
+        new_scylla_bin = self.params.get('update_db_binary')
+        if new_scylla_bin != '':
```

Comment:
None. I hit a problem and sent a patch to fix the problem in https://github.com/scylladb/scylla-longevity-tests/commit/57421b3790c30772ab41cefdcd3824c79d559a92